schemaregistry/serde/jsonschema/json_schema.go

```diff
@@ -0,0 +1,184 @@
+package jsonschema
+
+import (
+	"encoding/json"
+	"github.com/confluentinc/confluent-kafka-go/schemaregistry"
+	"github.com/confluentinc/confluent-kafka-go/schemaregistry/serde"
+	"github.com/invopop/jsonschema"
+	jsonschema2 "github.com/santhosh-tekuri/jsonschema/v5"
+	"io"
+	"strings"
+)
+
+// Serializer represents a JSON Schema BaseSerializer
+type Serializer struct {
+	serde.BaseSerializer
+	validate bool
+}
+
+// Deserializer represents a JSON Schema BaseDeserializer
+type Deserializer struct {
+	serde.BaseDeserializer
+	validate bool
+}
+
+var _ serde.Serializer = new(Serializer)
+var _ serde.Deserializer = new(Deserializer)
+
+// NewSerializer creates a JSON BaseSerializer for generic objects
+func NewSerializer(client schemaregistry.Client, conf *serde.Config, serdeType serde.Type, validate bool) (*Serializer, error) {
```

Comment:
I think there should be a different config type for each of proto / avro / json. And there should be a different type for serializers and deserializers. No superfluous properties. Can derive from serde.Config for commonality.

Reply:
even if not needed now, for extensibility.

Reply:
Ok, fixed